packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -317,13 +323,13 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
       }
       _extent._currentExtent.value = _extent.initialExtent;
     }
+    _child = widget.builder(context, _scrollController);
```

Comment:
What if the DraggableScrollableSheet is given a new `builder` during its lifetime?

Reply:
Unrelated: Looks like changes to `minChildSize` and `maxChildSize` are also not respected, which seems like another bug?

Reply:
I think I can address this with `didUpdateWidget` - will push something momentarily.

Reply:
In this case, that's too much - so I'm trying to cache the output of that builder to avoid rebuilding it whenever the sheet is dragged up or down.

This is simlar to how we don't rebuild every child sliver when a listview scrolls.

Reply:
Sounds good.